api/Apis/MetastoresApi.md

```diff
@@ -0,0 +1,33 @@
+# MetastoresApi
+
+All URIs are relative to *http://localhost:8080/api/2.1/unity-catalog*
+
+| Method | HTTP request | Description |
+|------------- | ------------- | -------------|
+| [**summary**](MetastoresApi.md#summary) | **GET** /metastore_summary | Get the summary of a metastore |
```

Comment:
Let's standardize on `Get a metastore summary` here and below.

Reply:
Fixed this (sorry had missed running build/sbt generate)